### ISSUE TITLE
Capture Additional data (req) when console.error occurs

### ISSUE THREAD

**Issue by bugb** [mentos1386/nest-raven#24]
## Package + Version

- [ ] `@sentry/node`
- [ ] other: `nest-raven`

### Version:

```
latest
```

## Description
When my code throw error, I can get the `ADDITIONAL DATA` with `req` field as you can see it here:

 
![image](https://user-images.githubusercontent.com/9280412/68358452-25687880-014b-11ea-8ca2-3acce32cbf66.png)

But when my code  does not throw anything and it use `console.error` instead, it only show something like this:

![image](https://user-images.githubusercontent.com/9280412/68358549-6bbdd780-014b-11ea-8f1d-6fb913878a11.png)

My Sentry setup:
![image](https://user-images.githubusercontent.com/9280412/68358657-c1927f80-014b-11ea-95fb-580a4c9e2425.png)

Could you please give me an approach get the `ADDITIONAL DATA` with `req` field like the first image?

**Comment by mentos1386** [comment]
Hmm, that looks like it's related to #22. Which means that nest-raven doesn't expose `ADDITIONAL DATA` to exceptions not captured by nest-raven itself. When you use `console.error` this isn't handled by `nest-raven` so no additional data is provided.

**Comment by bugb** [comment]
@mentos1386 Do you have a plan to fix this issue, I can use sample a hack like this:
```
// Inside  a controller
try {
  something();
} catch (e) {
  console.error(e.message, req) // here req is the request that is sent
}
```

But it seem it is not a good idea, what do you think?

**Comment by mentos1386** [comment]
The plan is for this to be fixed with #22.

Otherwise. It's a good practice to throw/faill when error happens. And not just logging it to stdout/stderr.
Then you can handle that exception with exception filter https://docs.nestjs.com/exception-filters.

**Comment by bugb** [comment]
@mentos1386 Thank you for your plan. Throw error is a good practice but for me I am maintaining a project that has many messy code with no document, sometimes I just want to capture events when something occur with `console.warn` and `console.error` instead of throw error, imagine you have an app solving vehicle routing problem with time windows, it is has many situations and results.

**Comment by mentos1386** [comment]
@bendadaniel that is provided by `@sentry/integrations` package. You can read more here https://getsentry.github.io/sentry-javascript/classes/_sentry_integrations.CaptureConsole.html.

Closing this as original issue would be addressed by #22.
